curvefs/src/metaserver/s3/metaserver_s3.h

```diff
@@ -36,6 +36,9 @@ class S3Client {
     virtual void Init(const curve::common::S3AdapterOption& option) = 0;
     virtual int Delete(const std::string& name) = 0;
     virtual int DeleteBatch(const std::list<std::string>& nameList) = 0;
```

Comment:
I'm wondering why we have this `S3Client` because it's just a wrapper of `S3Adapter`, and what it does is forward the function call to S3Adapter.

Reply:
> I'm wondering why we have this `S3Client` because it's just a wrapper of `S3Adapter`, and what it does is forward the function call to S3Adapter.

yep